xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -999,22 +990,10 @@ bool CDVDVideoCodecFFmpeg::GetPictureCommon(VideoPicture* pVideoPicture)
     pVideoPicture->color_range = 0;
 
   int qscale_type;
```

Comment:
since this was merged all Wayland builds are failed because reaching warning threshold.
this introduces a new warning
```
DVDVideoCodecFFmpeg.cpp:992, kodi compiler warnings, Priority: Normal
--
unused variable ���qscale_type��� [-Wunused-variable]
```

Reply:
variable is removed now

Reply:
thanks a lot.